tracer/src/Datadog.Trace/OTelMetrics/MetricPoint.cs

```diff
@@ -51,6 +75,14 @@ internal void UpdateCounter(double value)
         }
     }
 
+    internal void UpdateObservableCounter(double currentValue)
+    {
+        lock (_histogramLock)
+        {
+            _runningDoubleValue = currentValue;
```

Comment:
```suggestion
            _runningDoubleValue += currentValue;
```

Is it supposed to be an `=` to replace or is it supposed to be a `+=` like the `UpdateCounter` above?

Still going through the PR so this may be obvious later

Reply:
Maybe I just don't understand what "Observable" means